### DISCUSSION TITLE
trouble working with CMake and

### DISCUSSION THREAD

**Discussion by ramonicv** [LazyVim/LazyVim#2388] [Q&A]
I'm trying to make a full switch from vscode to nvim now, and lazyvim is great for that! However, I'm having trouble figuring out how to get CMake functionality and Header/Implementation file generation.

I got **cmake-tools.nvim** installed, but whenever I try to execute `:CMakeRun` I get this output:
"Cannot find CMakeLists.txt at cwd"

Not sure how to go about resolving this.

Another thing is that I was hoping to get header/implementation file generation using **clangd_extensions.nvim** , but am not sure how to go about it. I know there's the plugin to switch Source/Header for C/C++ with `<leader>cR`, but I don't know if I'm using it wrong or it just doesn't work for file generation. It always outputs "Corresponding file cannot be found"

please help

**Comment by serendipitywgy** [comment]
This problem is because you have not configured cmake_tools.nvim. You can carefully read the warehouse description of cmake_tools.nvim.
